jsapp/js/account/usage.component.tsx

```diff
@@ -1,58 +1,42 @@
 import {when} from 'mobx';
 import React, {useEffect, useMemo, useState} from 'react';
 import {useLocation} from 'react-router-dom';
-import type {AccountLimit, RecurringInterval} from 'js/account/stripe.api';
-import {getAccountLimits, getSubscriptionInterval} from 'js/account/stripe.api';
+import type {AccountLimit} from 'js/account/stripe.api';
+import {getAccountLimits} from 'js/account/stripe.api';
 import subscriptionStore from 'js/account/subscriptionStore';
 import LoadingSpinner from 'js/components/common/loadingSpinner';
 import UsageContainer from 'js/components/usageContainer';
 import envStore from 'js/envStore';
-import {formatDate, truncateNumber} from 'js/utils';
-import {getUsageForOrganization} from './usage.api';
+import {formatDate} from 'js/utils';
 import styles from './usage.module.scss';
 import LimitNotifications from 'js/components/usageLimits/limitNotifications.component';
+import useWhenStripeIsEnabled from 'js/hooks/useWhenStripeIsEnabled.hook';
+import {UsageContext, useUsage} from 'js/account/useUsage.hook';
 
-interface UsageState {
-  storage: number;
-  submissions: number;
-  transcriptionMinutes: number;
-  translationChars: number;
+interface LimitState {
   storageByteLimit: number | 'unlimited';
   nlpCharacterLimit: number | 'unlimited';
   nlpMinuteLimit: number | 'unlimited';
   submissionLimit: number | 'unlimited';
-  trackingPeriod: RecurringInterval;
-  currentMonthStart: string;
-  currentYearStart: string;
-  isUsageLoaded: boolean;
-  isLimitsLoaded: boolean;
-  isSubscriptionLoaded: boolean;
+  isLoaded: boolean;
 }
 
 export default function Usage() {
```

Comment:
Please add a short `/**` comment ❤️ Having one would make it easier to understand why we render this in `MyProjectsRoute` etc.